### ISSUE TITLE
Find what is setting full_size_download_allowed values to be "1" instead of 1

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2#3077]
### Descriptive summary

@KevinJonesMeta discovered some aerial photos and sports media guides works that had full_size_download_allowed set but were not allowing the download. This appears to be because the value was `"1"` and not `1`. But the local authority and the scripts I used to assign the values should have set the value to just `1`. 

The edit form seems to interpret the value as True on load, but the indexer appears to be more strict and does not set anything as allowed unless it's a `1`.

So possibly subsequent edits in bulkrax caused the conversion to a string. 

Looking at the importer: https://oregondigital.org/importers/603?locale=en
and an entry from there: https://oregondigital.org/importers/603/entries/433304?locale=en

the value in the raw and parsed metadata look to be correct.

### Expected behavior

Values should be either `1` or `0`

### Related work

Link to related tickets or prior related work here.

### Accessibility Concerns

Add any information here to indicate any known or suspected accessibility issues for this ticket

**Comment by wickr** [comment]
I found that works that had the wrong value set and thus indexed that way, the `full_size_download_allowed_sim` was `"1"` but the `full_size_download_allowed_label_sim` was not set to `True`. And if the UI goes off the value of the `_label_sim` here that could be why it's not registering.

Querying solr for works where the value is set but the label is not:
```ruby
results = ActiveFedora::SolrService.query("full_size_download_allowed_sim:\"1\" AND -full_size_download_allowed_label_sim:\"True\"", fl: 'id', rows: 100000)
```

initially resulted in 32,879 works. I fixed the values for the SCARC works (6605) so now the total returned is 26,274. Not sure what percentage of the remaining ones are UO but I believe most of them are. @lsat12357
